source/docs/networking/programming-your-radio.rst

```diff
@@ -0,0 +1,304 @@
+however they are 12V radios. Wire the radio to the 12V 2A terminals on
+the VRM (center-pin positive).**
+
+**Note: Teams will need to update firmware on both OM5P-AN and OM5P-AC
+radios in order for the programming utility to program them, or for them
+to be used at events. This must be done before you attempt to program
+them.**
+Pre-Requisites
+--------------
+
+The FRC Radio Configuration Utility requires the Java Runtime Engine
+(JRE). If you do not have Java installed, you can download the JRE from
+here: https://www.java.com/en/download/
+
+The FRC Radio Configuration Utility requires Administrator privileges to
+configure the network settings on your machine. The program should
+request the necessary privileges automatically (may require a password
+if run from a non-Administrator account), but if you are having trouble,
+try running it from an Administrator account.
+Application Notes
+-----------------
+
+By default, the Radio Configuration Utility will program the radio to
+enforce the 4Mbps bandwidth limit on traffic exiting the radio over the
+wireless interface. In the home configuration (AP mode) this is a total,
+not a per client limit. This means that streaming video to multiple
+clients is not recommended.
+
+The Utility has been tested on Windows 7, 8 and 10. It may work on other
+operating systems, but has not been tested.
+Programmed Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: images/pyr1.png
+   :width: 600
+   
+The Radio Configuration Utility programs a number of configuration
+settings into the radio when run. These settings apply to the radio in
+all modes (including at events). These include:
+
+-  Set a static IP of 10.TE.AM.1
+-  Set an alternate IP on the wired side of 192.168.1.1 for future
+   programming
+-  Bridge the wired ports so they may be used interchangeably
+-  The LED configuration noted in the graphic above.
+-  4Mb/s bandwidth limit on the outbound side of the wireless interface
+   (may be disabled for home use)
+-  QoS rules for internal packet prioritization (affects internal buffer
+   and which packets to discard if bandwidth limit is reached). These
+   rules are Robot Control and Status (UDP 1110, 1115, 1150) >> Robot
+   TCP & Network Tables (TCP 1735, 1740) >> Bulk (All other traffic).
+   (disabled if BW limit is disabled)
+-  DHCP server enabled. Serves out 10.TE.AM.11 - 10.TE.AM.111 on the
+   wired side, 10.TE.AM.130 - 10.TE.AM.230 on the wireless side, subnet
+   mask of 255.255.255.0, broadcast address 10.TE.AM.255
+-  DNS server enabled. DNS server IP and domain suffix (.lan) are served
+   as part of the DHCP.
+
+At home only:
+
+-  SSID may have a "Robot Name" appended to the team number to
+   distinguish multiple networks.
+-  Firewall option may be enabled to mimic the field firewall rules
+   (open ports may be found in the Game Manual)
+
+**Note: It is not possible to modify the configuration manually**
```

Comment:
Replace with 
`.. note:: Note: It is not possible to modify the configuration manually`